recipe-server/client/control/components/MultiPicker.js

```diff
@@ -51,24 +42,16 @@ class PickerControl extends React.Component {
    * @return {Array<Object>}  Array of available options that meet search criteria
    */
   getFilteredOptions(options, search) {
-    let newOptions = [].concat(options);
-
-    // There may not even be a search, so check that first.
     if (search) {
       // Lowercase-ify the search value to remove case sensitivity
       const filterValue = search.toLowerCase();
 
-      // For each option, create a string value to search over, and then determine
-      // if that value contains our search text.
-      newOptions = newOptions.filter(({ value, label }) => {
-        const searchValues = [value, label].join('   ').toLowerCase();
-        return searchValues.indexOf(filterValue) > -1;
-      });
+      // Determine if selected option properties contain the filterValue.
+      return options.filter(({ value, label }) =>
+        [value, label].find(str => str.toLowerCase().indexOf(filterValue) > -1));
```

Comment:
[`Array.prototype.some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) would be more appropriate here than just `find`, but they both return truthy responses, so it's just a nit.